src/main/java/in/projecteka/consentmanager/clients/ConsentArtefactNotifier.java

```diff
@@ -16,6 +20,10 @@
 public class ConsentArtefactNotifier {
     private final WebClient.Builder webClientBuilder;
     private final Supplier<Mono<String>> tokenGenerator;
+    private final GatewayServiceProperties gatewayServiceProperties;
+
+    private static final String HDR_HIP_ID = "X-HIP-ID";
```

Comment:
Check whether it's used in multiple places, please create a consent file and use it everywhere.